### DISCUSSION TITLE
Specify a default stable diffusion checkpoint model

### DISCUSSION THREAD

**Discussion by shreya-dama-Crafty-Apes-VFX** [AbdBarho/stable-diffusion-webui-docker#427] [General]
Is there a way to specify a checkpoint model that I want as a default load model in the dockerfile?

<img width="401" alt="image" src="https://user-images.githubusercontent.com/117180689/235000917-a53b0cc5-1563-4633-ac49-3ba72cce4b79.png">

Stable Diffusion folder:
<img width="352" alt="image" src="https://user-images.githubusercontent.com/117180689/235001290-dd631805-a892-48b0-8f42-b1f584c5ed9d.png">

**Comment by AbdBarho** [comment]
which UI?

**Reply by shreya-dama-Crafty-Apes-VFX** [reply]
A1111

**Comment by digger73** [comment]
Are you creating a new docker container every time ? 
Cause my docker container remembers what I used the last time, so I just spin up my container instead of creating a new.

**Reply by shreya-dama-Crafty-Apes-VFX** [reply]
Yes, I am creating a new container everytime to make sure it doesn't take up the memory when not in use(I load up the image from a .tar file and the create container using docker run)
